tests/core/snap-repair/uc24-run.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -ex
```

Comment:
Ditto

Reply:
the only problem about this is we have many test scripts with the same problem. I suppose it could be included in a following pr, also we could update this for other script in other tets.